### DISCUSSION TITLE
Zustand Persist cause Hydration Failed.

### DISCUSSION THREAD

**Discussion by eitai** [pmndrs/zustand#1603] [Q&A]
hey guys, I try to solve a complicated Bug that we suffering from, as it seems is that Zustand Persist is causing Hydration failure because local storage is not exist in serverside  in Next.js

![image](https://user-images.githubusercontent.com/35004992/216936014-c8b852d3-2c44-43ec-aaf0-a1dff9c38bbc.png)

**Comment by dai-shi** [comment]
Does #1537 seem related?
